src/HotChocolate/ElasticSearch/test/Data.ElasticSearch.Tests/__snapshots__/StringPrefixTests.ElasticSearch_MultipleField.snap

```diff
@@ -0,0 +1,12 @@
+﻿{
```

Comment:
query?

Reply:
Similarly, the query for this test should be found under `__snapshots__/StringPrefixTests.ElasticSearch_MultipleField_query.snap`.

Reply:
Cool :) Just overlooked it apparently :)